### PR TITLE
update setup.py

### PR DESCRIPTION
**Pull request by nickmoreton**
# Create package file 

This PR only changes the version reference to 0.1.3 and updates the documentation for the install process.
It's now been rebased on the current main branch because it was out of date.

The package has been uploaded to PyPi: https://pypi.org/project/wagtail-wordpress-import/

Ticket URL: https://projects.torchbox.com/projects/wordpress-to-wagtail-importer-package/tickets/61

---

- Testing
    - [x] CI passes
    - [x] If necessary, tests are added for new or fixed behaviour
    - [x] These changes do not reduce test coverage
- Documentation.
    - [x] This PR adds or updates documentation
- Import.
    - [x] Tested against: https://raw.githubusercontent.com/WPTT/theme-unit-test/master/themeunittestdata.wordpress.xml with one caveat: In the xml is one page item for url https://wpthemetestdata.wordpress.com/2009/09/05/edge-case-no-title/ which has no title. Removing this fixture is required. We always expect a page title.

### CODE REVIEW

File: wagtail_wordpress_import/importers/wordpress.py [hunk 1]
```diff
@@ -304,6 +303,23 @@ def connect_page_categories(self, page, category_model, item):
             page.categories = page_categories
 
 
+def default_prefilters():
```
Comment (nimasmi) [inline wagtail_wordpress_import/importers/wordpress.py:321]: 1. **Comment**: As discussed elsewhere, I agree this should change from a function to a constant.
2. **Question**: I also commented about functions vs constants at https://github.com/torchbox/wagtail-wordpress-import/pull/106. I'm confused about which conversation was which. Was I wrong there?

**Review by nimasmi — CHANGES_REQUESTED** [review]
There is one change requested, as discussed elsewhere.

**Review by nimasmi — APPROVED** [review]

**Comment by nimasmi** [comment]
In future, you can use https://test.pypi.org/ for testing package installation and the use of package managers, and then you don't need to bump the version number for the public facing package on the live PyPI.
